api/client/client.go

```diff
@@ -3359,27 +3360,6 @@ func (c *Client) GetDatabaseObjectImportRules(ctx context.Context) ([]*dbobjecti
 	return out, nil
 }
 
-// GetDatabaseObjects retrieves all database objects.
-func (c *Client) GetDatabaseObjects(ctx context.Context) ([]*dbobjectv1.DatabaseObject, error) {
```

Comment:
Is this used anywhere? 

`api` module should be stable

Reply:
Technically this is a breaking change, but the feature is very new and I doubt this will impact anyone.

Reply:
I've removed `GetDatabaseObjects` as it wasn't used in our code, but it isn't hard to reintroduce; just done so.

Reply:
I've restored the public API.